config/data_funcs.go

```diff
@@ -156,13 +157,17 @@ func unmarshalOrReturnString(value string) (result interface{}) {
 
 func parseCmdline(cmdLine string) map[interface{}]interface{} {
 	result := make(map[interface{}]interface{})
+	params, err := shellwords.Parse(cmdLine)
```

Comment:
Can you switch this to not use the global parser and explicitly set `ParseEnv` and `ParseBackticks` to false? If the global defaults were ever changed in go-shellwords it might cause some bad results.

```
p := shellwords.NewParser()
p.ParseEnv = false
p.ParseEnv = false
params, err := p.Parse(cmdLine)
```